src/sentry/search/events/datasets/discover.py

```diff
@@ -677,6 +677,109 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
+                                                                                            [
+                                                                                                Function(
+                                                                                                    "equals",
+                                                                                                    [
+                                                                                                        Identifier(
+                                                                                                            "x"
+                                                                                                        ),
+                                                                                                        args[
+                                                                                                            "spans_op"
+                                                                                                        ],
+                                                                                                    ],
+                                                                                                ),
+                                                                                                Function(
+                                                                                                    "equals",
+                                                                                                    [
+                                                                                                        Identifier(
+                                                                                                            "y",
+                                                                                                        ),
+                                                                                                        args[
+                                                                                                            "spans_group"
+                                                                                                        ],
+                                                                                                    ],
+                                                                                                ),
+                                                                                            ],
+                                                                                        ),
+                                                                                    ),
+                                                                                    Column(
+                                                                                        "spans.exclusive_time_32"
```

Comment:
Snuba exposes `spans.exclusive_time` as an alias to the 32 bit column. So no need to be explicit about it here and let snuba take care of this implementation detail.

```suggestion
                                                                                        "spans.exclusive_time"
```

Reply:
ah, kk good to know 👍 done